### PR TITLE
Add layout test

### PR DESCRIPTION
**Pull request by acatherwood**
Testing 1,2,3

### CODE REVIEW

File: app/src/main/java/com/standuptracker/MainActivity.kt [hunk 1]
```diff
@@ -36,8 +36,7 @@ class MainActivity : AppCompatActivity() {
         // menu should be considered as top level destinations.
         appBarConfiguration = AppBarConfiguration(
             setOf(
-                R.id.nav_home, R.id.nav_gallery, R.id.nav_slideshow,
-                R.id.nav_tools, R.id.nav_share, R.id.nav_send
+                R.id.nav_home, R.id.nav_gallery
```
Comment (acatherwood) [inline app/src/main/java/com/standuptracker/MainActivity.kt:39]: Search by date functionality looks good

File: app/src/main/res/layout/fragment_home.xml [hunk 2]
```diff
@@ -16,4 +17,33 @@
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toTopOf="parent" />
+
+    <EditText
+        android:id="@+id/txtDate"
```
Comment (acatherwood) [inline app/src/main/res/layout/fragment_home.xml:22]: thanks for renaming this textbox!

File: app/src/main/java/com/standuptracker/MainActivity.kt (hunk 1 above)
Comment (elkhalci) [reply app/src/main/java/com/standuptracker/MainActivity.kt:39]: Thank you

File: app/src/main/res/layout/fragment_home.xml (hunk 2 above)
Comment (elkhalci) [reply app/src/main/res/layout/fragment_home.xml:22]: I think txtDate is appropriate name as we need to retrieve standup notes by date /day.

**Pull request merged**